### ISSUE TITLE
loss unchanged after about 200 epochs

### ISSUE THREAD

**Issue by backkon** [polarisZhao/PFLD-pytorch#53]
Hello, I trained PFLD according to your code. The loss on the validation dataset is unchanged after about 200 epochs, and finally nme is 0.074. Why is this? Looking forward to your reply~

**Comment by Incandescenced1** [comment]
me,too.The eval loss is just 0.2 after about 200 epochs
